src/rpc/misc.cpp

```diff
@@ -1431,6 +1488,105 @@ UniValue getsparklatestcoinid(const JSONRPCRequest& request)
     return UniValue(latestCoinId);
 }
 
+UniValue getmempooltxids(const JSONRPCRequest& request)
```

Comment:
This name must have "spark" in it. Otherwise, someone will find this RPC and complain that it's not working as he wants it to. As a matter of fact, all new RPCS should follow the same rule

Reply:
agree, updated